src/styles/product_card.scss

```diff
@@ -72,8 +72,9 @@ $cards-distance: 144px;
 
   &__value {
     font-size: $primaryFont-size;
+    font-weight: 400;
     line-height: 13px;
-    align-self: flex-end;
+    margin-top: 5px;
```

Comment:
```suggestion
```

Reply:
you can set `align-items: baseline;` to a container. instead of using magic numbers.